math_hurdlerActivity.py

```diff
@@ -90,4 +90,5 @@ def read_file(self, file_path):
         self.game.read_file(file_path)
 
     def write_file(self, file_path):
-        self.game.write_file(file_path)
+        # self.game.write_file(file_path)
```

Comment:
I'm guessing you forgot about this.

Reply:
As of now, the game does not resume, it restarts. I'll get it right soon.

Reply:
When you say it restarts, I'm guessing you mean you're yet to implement saving to journal correctly so when the activity instance resumes no values have been stored to be restored so instead you just start a new instance, is this correct?

Reply:
Yes, I did temporarily until that feature is ready.

Reply:
Should I add this feature in this PR or another PR?

Reply:
You can add it here after fixing the existing issues here.

Reply:
done